blocks/init/src/blocks/components/hamburger/assets/hamburger.js

```diff
@@ -0,0 +1,51 @@
+import { getNavigatorVibrate } from './navigator';
+
```

Comment:
please rewrite this as a class so it is consistent with our other JS classes

Reply:
example: https://github.com/infinum/wordpress-infinum-web/tree/staging/wp-content/themes/infinum/src/blocks/components/header/assets

Reply:
Ok but we should rethink this for JS components that don't use state, there's no need to make it a class.

Reply:
They'd also make sense if we used typescript - which we don't.

Reply:
We are writing classes for everything in JS (it is ok for some full simple stuff). Because it is easier to extend/reuse, organise, etc the code. In general all benefits of using Classes in PHP relates in JS also